ext/opentelemetry-ext-mysql/tests/test_mysql_integration.py

```diff
@@ -41,4 +31,6 @@ def test_trace_integration(self):
             cursor = cnx.cursor()
             query = "SELECT * FROM test"
             cursor.execute(query)
-            self.assertTrue(start_as_current_span.called)
+            spans_list = self.memory_exporter.get_finished_spans()
+            self.assertEqual(len(spans_list), 1)
+            # TODO: Add more tests?
```

Comment:
Are more tests needed here? I was under the impression that most of the functionality is tested in the integration tests. I would suggest opening an issue to track this if it's the case, otherwise I fear we'll just forget about it.

Reply:
I wasn't aware of the integration tests at the time I wrote that comment. I just removed it.